src/pgduckdb_options.cpp

```diff
@@ -178,36 +178,36 @@ DuckdbCacheObject(Datum object, Datum type) {
 
 extern "C" {
 
-PG_FUNCTION_INFO_V1(install_extension);
+DECLARE_PG_FUNCTION(install_extension);
+DECLARE_PG_FUNCTION(pgduckdb_raw_query);
+DECLARE_PG_FUNCTION(cache);
+DECLARE_PG_FUNCTION(pgduckdb_recycle_ddb);
```

Comment:
nit: it seems nice to keep these close to the declaration like they were before

Reply:
It might even be nice to take the macro one step further end have the definition be like the below instead. Because it is a bit confusing that here we declare the function with the `_cpp` suffix but then the actual function uses the `_cpp` suffix. I think it would be nice to hide that implementation detail:

```c
DECLARE_PG_FUNCTION(install_extension) {
	Datum extension_name = PG_GETARG_DATUM(0);
	bool result = pgduckdb::DuckdbInstallExtension(extension_name);
	PG_RETURN_BOOL(result);
}
```